pyproject.toml

```diff
@@ -0,0 +1,31 @@
+# NOTE: You have to use single-quoted strings in TOML for regular expressions.
+# It's the equivalent of r-strings in Python. Multiline strings are treated as
+# verbose regular expressions by Black. Use [ ] to denote a significant space
+# character.
+
+[tool.black]
+include = '\.pyi?$'
+line-length = 120
+py36 = false
+skip-string-normalization = true
+exclude = '''
+(
+  /(
+    | \.git
```

Comment:
not sure the below list is ok

Reply:
I think we can instead include only `'tasks` folder, we have python code only there

Reply:
I launch linters only on `tasks` repo, so no need to `exclude/include`